spec/unit/govuk_index/presenters/indexable_content_presenter_spec.rb

```diff
@@ -115,36 +115,16 @@
     end
   end
 
-  context "transaction with hidden_search_terms (smart answer start page)" do
-    let(:format) { "transaction" }
+  context "smart_answer format" do
+    let(:format) { "smart_answer" }
     let(:details) do
       {
-        "introductory_paragraph" => [
-          {
-            "content" => "intro\n",
-            "content_type" => "text/govspeak",
-          },
-          {
-            "content_type" => "text/html",
-            "content" => "<p>intro</p>\n",
-          },
-        ],
-        "more_information" => [
-          {
-            "content" => "more<\n",
-            "content_type" => "text/govspeak",
-          },
-          {
-            "content_type" => "text/html",
-            "content" => "<p>more</p>\n",
-          },
-        ],
         "hidden_search_terms" => ["hidden 1", "hidden 2"],
       }
     end
 
     it "hidden_search_terms is correctly indexed" do
-      expect(subject.indexable_content).to eq("hidden 1\n\n\nhidden 2\n\n\nintro\n\n\n\nmore")
+      expect(subject.indexable_content).to eq("hidden 1\n\n\nhidden 2")
```

Comment:
Is there any value adding that body and more_information data in here as an explicit example or is it enough to just keep testing that there's something in there?

Reply:
These tests seem to be testing that **_at least_** the correct information is been included in the `indexable_content` method and assume that `details` represented are accurate for the content schema. I guess there might be value in making sure "other" fields from from details aren't being included, but I've just written this to be consistent with the other tests.